### PR TITLE
[SYNCOPE-1439] Fixed user membership attributes not updating from Admin Console

### PR DESCRIPTION
**Pull request by mat-ale**
Fix for issue [SYNCOPE-1439](https://issues.apache.org/jira/browse/SYNCOPE-1439) on version `2.0.x`.

### CODE REVIEW

File: client/console/src/main/java/org/apache/syncope/client/console/wizards/any/PlainAttrs.java [hunk 1]
```diff
@@ -112,7 +115,7 @@
 
             @Override
             public WebMarkupContainer getPanel(final String panelId) {
-                return new PlainSchemas(panelId, schemas, attrTOs);
+                return new PlainSchemasSimple(panelId, schemas, attrTOs);
```
Comment (ilgrosso) [inline client/console/src/main/java/org/apache/syncope/client/console/wizards/any/PlainAttrs.java]: Why not name `PlainSchemasSimple` as `PlainSchemasOwn` (considering that the other one is named `PlainSchemasMemberships`)?

File: client/console/src/main/java/org/apache/syncope/client/console/wizards/any/PlainAttrs.java [hunk 2]
```diff
@@ -363,15 +375,91 @@ protected PageReference getPageReference() {
         return panel;
     }
 
-    public class PlainSchemas extends Schemas {
+    public class PlainSchemasMemberships extends PlainSchemas<AttributableTO> {
```
Comment (ilgrosso) [inline client/console/src/main/java/org/apache/syncope/client/console/wizards/any/PlainAttrs.java]: Any point to have this class and its sister class as public? Why not `protected static`, or at least `protected` if needed?

File: client/console/src/main/java/org/apache/syncope/client/console/wizards/any/PlainAttrs.java (hunk 1 above)
Comment (mat-ale) [reply client/console/src/main/java/org/apache/syncope/client/console/wizards/any/PlainAttrs.java]: Good point, I'll do that thanks!

File: client/console/src/main/java/org/apache/syncope/client/console/wizards/any/PlainAttrs.java (hunk 2 above)
Comment (mat-ale) [reply client/console/src/main/java/org/apache/syncope/client/console/wizards/any/PlainAttrs.java]: Yes `protected` seems better!

**Pull request merged**